### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by vmwhoami**
Setup the tests for my project.
Added unit tests for your calculate.js file.
Elements are displayed at:
Home page
Calculator page
Quote page by using jest snapshots.

Sorry about the last pull request I was in a hurry and didn't bother to check the branch

### CODE REVIEW

File: src/__test__/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  expect(total).toBe(-12);
+  expect(next).toBe(-6);
+  expect(operation).toBe('+');
+});
+
+test('+/-  should make all number negative', () => {
+  const obj = Calculate(object, '%');
+  const { total } = obj;
+  expect(total).toBe(0.12);
+});
```
Comment (krys2fa) [inline src/__test__/calculate.test.js]: - [ ] [REQUIRED] Even though the tests here are different, the test descriptions are the same. Please correct this.

File: src/__test__/calculate.test.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  const { total } = obj;
+  expect(total).toBe('6');
+});
+
+test('Calculate should calculate subsctracion', () => {
+  const obj = Calculate(mult, '=');
+  const { total } = obj;
+  expect(total).toBe('72');
+});
+
```
Comment (krys2fa) [inline src/__test__/calculate.test.js]: - [ ] [REQUIRED] Even though the tests here are different, the test descriptions are the same. Please correct this.

File: src/logic/calculate.js [hunk 3]
```diff
@@ -60,12 +81,9 @@ const Calculate = (dataObj, btnName) => {
       next = null;
     }
   }
+  // convToString(total, next, operation)
```
Comment (krys2fa) [inline src/logic/calculate.js]: - [ ] [REQUIRED]  It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

**Review by krys2fa — CHANGES_REQUESTED** [review]
## Nicely done @vmwhoami  👏 
You wrote tests covering both components and logic. 💪 

### Status: Changes Required ♻️ 
Check the comments below for the requested changes.

Please request another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

**Pull request merged**